proposals/p2922.md

```diff
@@ -71,26 +78,87 @@ Our current CoC team:
 
 ## Proposal
 
-Recruit and train 5+ potential new conduct team members to deal with
-conduct-related escalations on behalf of and for the Carbon community, and
-replace the current CoC team as soon as enough members are ready.
+Create a new [Conduct Team](/docs/project/teams/conduct_team.md) as follows:
+
+-   Recruit and train 5+ potential new conduct team members to deal with
+    conduct-related escalations on behalf of and for the Carbon community, and
+    replace the current CoC team as soon as enough members are ready.
+-   We should always have at least 3 conduct team members available to respond
+    promptly, even factoring in vacations, getting sick, or other normal
+    disruptions.
+-   Subsequent updates to the exact members will be managed by the new conduct
+    team themselves, and without full proposals but just PRs to update the
+    relevant documentation.
+
+Also remove a restriction on how reports are shared with new conduct team
+members going forward (but _not_ retroactively, which would violate our past
+promise). While this restriction was well intentioned, it isn't a restriction
+that the conduct team can effectively uphold going forward. For example, it
+would prevent informing new conduct team members of people banned because of
+bad-faith reports. We understand that reports are sensitive, and have
+strengthened our statements around keeping them confidential and only using them
+as needed.
 
-We should always have at least 3 conduct team members available to respond
-promptly, even factoring in vacations, getting sick, or other normal
-disruptions.
+## Details
 
-Subsequent updates to the exact members will be managed by the new conduct team
-themselves, and without full proposals but just PRs to update the relevant
-documentation.
+### Strategy for a sustainable Conduct Team
 
-## Details
+The Carbon community needs a **distinct, qualified, empowered, and diverse**
+Code of Conduct team.
 
-### Initial team members
+This crucial and particularly straining work needs to be shared among the team,
+so all members get enough support and rest. The proposal is for a member
+rotation strategy. Carbon's long-term goal should be to have 5+ trained Conduct
+Team members, so at least 3 are available to discuss and act upon incident
+reports in a timely and adequate manner.
+
+The initial team was recruited by our Community lead and pulls from the current
+trained moderator pool. We propose that the Conduct team itself is responsible
+to continue recruiting both new Conduct team members and new moderators to make
+both efforts sustainable. They will work toward better representation, embracing
+both broad and intersectional diversity.
+
+### What we are looking for in Conduct team members
```

Comment:
I don't think thot proposals are *strictly* one-off? They just aren't explicitly maintained *living* documents.

I can try to also create some maintained, living documentation for how the conduct team, but just putting it into `conduct_team.md` seems a bit awkward. And I'm hesitant to drive creating the living documentation for the conduct team as I'm not going to be *on* that team going forward.

For language features, we routinely create a proposal first, and then incorporate it as needed into the living design document. If, when, and where the conduct team wants to create a persistent document that incorporates this, I think that would be OK to do in follow-up PRs, and those wouldn't need to be proposals.

For now, rather than spending time figuring out the right structure and wording to make these effective as on-going living documentation, I'd prefer to start here.

Reply:
The rule of thumb that I prefer is that any information that has ongoing, non-historical relevance should live somewhere other than a proposal document, both because proposals aren't living documents and because the information in them isn't very discoverable.

However, you're right that we can (and often do) accomplish that with follow-up PRs, and you make a good point that the conduct team should probably take the lead on those documents, so I'm OK with deferring it for now.

Reply:
Double checked with the leads to make sure they're comfortable with this plan, and they are so we should be good to land this as is and then any improvements to more long-term docs can be follow-up work led by the conduct team themselves.